apple/internal/xcframework_rules.bzl

```diff
@@ -864,6 +864,11 @@ built for those platform variants (for example, `x86_64`, `arm64`) as their valu
 A dictionary of strings indicating which platform variants should be built for the tvOS platform (
 `device` or `simulator`) as keys, and arrays of strings listing which architectures should be
 built for those platform variants (for example, `x86_64`, `arm64`) as their values.
+""",
+            ),
+            "macos": attr.string_list(
+                doc = """
+A list of strings indicating which architecture should be built for the macOS platform (for example, `x86_64`, `arm64`).
```

Comment:
Keeping this a `string_list_dict` keeps the logic simpler, and maybe we just fail if folks pass `simulator`?